### DISCUSSION TITLE
Dans le page des Applications Exploitation et Editeur ?

### DISCUSSION THREAD

**Discussion by moiiom59** [dbarzin/mercator#563] [General]
Bonjour,
ne pourrait on pas utiliszer lameêm table pour la liste Exploitation et le liste Editeur qui est une zone libre alors que nos Editeurs sont identtifier comme les exploitants ?
merci à vous

**Comment by dbarzin** [comment]
Ce n'est pas toujours le cas. Par exemple M$ est rarement l'exploitant de ses solutions.

**Comment by moiiom59** [comment]
Bonjour,
Voici le log

Merci à vous

Bien cordialement,

**Comment by dbarzin** [comment]
Toujours pas. Peux-tu poster les logs sur GitHub et pas les envoyer par mail ?

**Comment by moiiom59** [comment]
C’est fait !
merci

Bien cordialement,

**Comment by dbarzin** [comment]
Corrigé  dans https://github.com/dbarzin/mercator/pull/566
